### DISCUSSION TITLE
still working on this?

### DISCUSSION THREAD

**Discussion by richrobber2** [TiagoMarinho/Soph#20] [General]
just wondering if u are still working on this project?

**Comment by TiagoMarinho** [comment]
Yes I am still working on this project. Right now I'm moving from Brazil to Germany so maybe it will take a few weeks until I can resume work on Soph, but once that's over development here is still going to be very active.

I hope my recent hiatus hasn't put out your interest in this project. Any issues or requests you may have please let me know.
